From 5f67ef56cc102b47311c154ed6a9bc8f54a4fb99 Mon Sep 17 00:00:00 2001
From: Stuart Hayhurst <stuart.a.hayhurst@gmail.com>
Date: Sat, 22 Jun 2024 20:25:21 +0100
Subject: [PATCH] hwdb: Fix Logitech G915 TKL (Bluetooth) appearing as a mouse

(cherry picked from commit 5e717db67c33f1dd55692da1dbba97c3743547db)

Related: RHEL-5950
---
 hwdb.d/60-input-id.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-input-id.hwdb b/hwdb.d/60-input-id.hwdb
index b9d1ce0fc0..dfb035de2a 100644
--- a/hwdb.d/60-input-id.hwdb
+++ b/hwdb.d/60-input-id.hwdb
@@ -59,6 +59,10 @@ id-input:modalias:input:b0003v28bdp0078*
 id-input:modalias:input:b0003v04B3p301Ee0100-e0,1,2,4*
  ID_INPUT_POINTINGSTICK=1
 
+# Logitech G915 TKL Keyboard (Bluetooth)
+id-input:modalias:input:b0005v046DpB35Fe0022*
+ ID_INPUT_MOUSE=0
+
 # Logitech Ultrathin Touch Mouse
 id-input:modalias:input:b0005v046DpB00De0700*
  ID_INPUT_MOUSE=1
